packages/ember-flight-icons/package.json

```diff
@@ -37,47 +37,49 @@
     "@hashicorp/flight-icons": "^2.20.0",
     "ember-auto-import": "^2.6.3",
     "ember-cli-babel": "^7.26.11",
-    "ember-cli-htmlbars": "^6.2.0"
+    "ember-cli-htmlbars": "^6.2.0",
+    "ember-get-config": "^2.1.1"
```

Comment:
@Dhaulagiri I tried reverting `dependency` updates to avoid releasing new versions for this update, but adding `ember-get-config` is still required so I'm not sure how we'll succeed in this

Reply:
ah, that should work – I'll give it a go